backends/backend.conf

```diff
@@ -4,6 +4,104 @@ backend {
   default = "Local"
   providers {
 
+    slurm_singularity {
+      actor-factory = "cromwell.backend.impl.sfs.config.ConfigBackendLifecycleActorFactory"
+      config {
+        script-epilogue = "sleep 30"
+        concurrent-job-limit = 50
+        runtime-attributes = """
+          Int cpu = 1
```

Comment:
What about if the user wants gpu? For example on sherlock they would ask for:

```
--partition gpu --gres gpu:1
```
or something like that. In this context, they would also need to add `--nv` as a `singularity_command_option`. This is ok because it's consistent, but I think it would be good to have this sort of use case / example front and center.

Reply:
will add support for gpu for SLURM/SGE.

Reply:
okay great! I can almost assure you (at least based on tickets we get) that everyone and their grandma-ma are going to want GPU, so this is greatly appreciated! It's also just confusing generally how to specify it (the whole `--gres:gpu1` or similar) deal so helping out with some sort of boolean is definitely helpful!